src/crypto-api/index.ts

```diff
@@ -589,6 +589,11 @@ export enum DecryptionFailureCode {
      */
     HISTORICAL_MESSAGE_USER_NOT_JOINED = "HISTORICAL_MESSAGE_USER_NOT_JOINED",
 
+    /**
+     * The sender does not satisfy the requested trust requirement.
```

Comment:
could you give a bit more context about what a "trust requirement" means? some reference to how that is requested, maybe?

Reply:
I've removed the error code for now, since it can't be triggered yet.  I'll re-add it (with a better comment) later when I add the functionality that allows this error to be triggered.